src/harness/reference_models/fss_purge/testdata/cbsd_1.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "device_1_id",
+	"registrationRequest": {
+					"fccId": "fccId1",
+		            "cbsdCategory": "B",
+		            "callSign": "KPPP",
+		            "userId": "cbsd_1",
+		            "airInterface": {
+    				  "radioTechnology": "E_UTRA",
+                      "supportedSpec": "LTE-Rel10"
+                    },
+		            "cbsdInfo": {
+                      "vendor": "Equipment Maker C"
+                    },
+
+					"cbsdSerialNumber": "serial_1",
+                    "measCapability": [ "EUTRA_CARRIER_RSSI_ALWAYS" ],
+                    "installationParam": {
+                      "latitude": 38.99527,
+                      "longitude": -77.45583,
+                      "height": 75,
+                      "heightType": "AGL",
+                      "indoorDeployment": false,
+                      "antennaAzimuth": 170,
+                      "antennaDowntilt": -5,
+                      "antennaGain": 0,
+                      "antennaBeamwidth": 60,
+                      "eirpCapability": 47
+                    }
+			    	},
+    "grantRequests": [{
+					"channelType": "GAA",
+					"grantExpireTime": "calculated at runtime",
+					"id": "cbsd_1_grant_0",
+					"operationParam": {
+						"maxEirp": 37,
+						"operationFrequencyRange": {
+							"highFrequency": 3692000000.0,
+							"lowFrequency": 3690000000.0
+						}
+					},
+					"terminated": false
+				},
+                                 {
+                                        "channelType": "GAA",
+                                        "grantExpireTime": "calculated at runtime",
+                                        "id": "cbsd_1_grant_1",
+                                        "operationParam": {
+                                                "maxEirp": 37,
+                                                "operationFrequencyRange": {
+                                                        "highFrequency": 3696000000.0,
+                                                        "lowFrequency": 3694000000.0
+                                                }
+                                        },
+                                        "terminated": false
+                                },
+                                   {
+                                        "channelType": "GAA",
+                                        "grantExpireTime": "calculated at runtime",
+                                        "id": "cbsd_1_grant_2",
+                                        "operationParam": {
+                                                "maxEirp": 37,
+                                                "operationFrequencyRange": {
+                                                        "highFrequency": 3698000000.0,
+                                                        "lowFrequency": 3696000000.0
+                                                }
```

Comment:
maxEirp should be with decimal points
Frequency should be integer

Reply:
fixed